development/markdown-how-to.md

```diff
@@ -0,0 +1,22 @@
+---
+navTitle: Markdown How-To
+---
+
+## Markdown Cheatlist
```

Comment:
Can't we just link to the spec: https://daringfireball.net/projects/markdown/syntax ?

Reply:
That's horribly formatted, and difficult to scan/parse for people new to the syntax

Reply:
Whilst this is great for the basics to get started, we don't want to be maintaining a full page of docs on markdown. Including a link to a more complete reference would be helpful - and there are plenty out there that are more accessible than Gruber's page.

 - https://www.markdownguide.org/cheat-sheet/
 - https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax

Reply:
To clarify - I suggest to include a link **in addition** to this high level cheat sheet of the basics.